pkg/envvar/envvar.go

```diff
@@ -14,14 +14,15 @@ const (
 )
 
 type Env struct {
-	AuthJwksAud      string `split_words:"true" required:"true"`
-	AuthJwksIss      string `split_words:"true" required:"true"`
-	AuthJwksUrl      string `split_words:"true" required:"true"`
-	ChainId          string `split_words:"true" required:"true"`
-	ChainRpcEndpoint string `split_words:"true" required:"true"`
-	HttpHost         string `split_words:"true" default:"127.0.0.1"`
-	HttpPort         string `split_words:"true" default:"7777"`
-	MarketsAddress   string `split_words:"true" required:"true"`
+	AuthJwksAud       string `split_words:"true" required:"true"`
+	AuthJwksIss       string `split_words:"true" required:"true"`
+	AuthJwksUrl       string `split_words:"true" required:"true"`
+	ChainId           string `split_words:"true" required:"true"`
+	ChainRpcEndpoint  string `split_words:"true" required:"true"`
+	HttpHost          string `split_words:"true" default:"127.0.0.1"`
+	HttpPort          string `split_words:"true" default:"7777"`
+	MarketsAddress    string `split_words:"true" default:"0xBa230f4Bf34E48D04e65dE9a0F6Fe5EcDAa0c17A"`
+	RandomizerAddress string `split_words:"true" default:"0x63f01b695c67B764e823F972bc61fcAFbac5102b"`
```

Comment:
Let's put the contract addresses somewhere common where we can define addresses per chain/environment.

Reply:
https://github.com/uvio-network/apiserver/pull/49/commits/14f019b05ad2ac3826e11e3be4c2770a198b6ddd. Didn't implement a common place to store the addresses yet. Will do so after i unblock https://github.com/uvio-network/issues/issues/5.